### ISSUE TITLE
Have a plan to support GCP(Ground control point)?

### ISSUE THREAD

**Issue by mac516** [colmap/colmap#767]
Hello,

Does COLMAP have a plan to support GCP (Ground control point) ?
GCP are landmarks visible on the images for which the geospatial position is known.
It is useful to align reconstructed points to accurate geospatial position. 
Other SfM tools support this function.

In this forum, someone already discussed about GCP. 
How about recent status?

Thank you.

-mac516

**Comment by martinVyoo** [comment]
Model_aligner should allow you to align your sparse reconstruction with reference points
However, it aligns the cameras, not arbitrary 3D points

**Comment by pmd3sallijr** [comment]
Yes I agree. Its a great bit of free software that just needs this adding!!

**Comment by ahojnnes** [comment]
There is no update and, unless there is a community contribution, I don't think I would have time to implement this feature anytime soon.

**Comment by jared-krauss** [comment]
Does this help at all? https://github.com/lcmrl/COLMAP_GroundControlPoints
